docs/src/testing/fork-testing.md

```diff
@@ -57,16 +61,11 @@ block_id.tag = "Latest"
 [[tool.snforge.fork]]
 name = "SOME_SECOND_NAME"
 url = "http://your.second.rpc.url"
-block_id.tag = "Pending"
+block_id.number = "123"
```

Comment:
I believe we have to change the parsing scarb.toml functions (not sure which ones) because allow passing `block_id.tag = "Pending"` at the moment.
Afaik @piotmag769 refactored it recently, so he can have the best knowledge about it

Reply:
Yep, you can change it when #919 is merged in `crates/forge/src/scarb/config.rs` and in `crates/test-collector/src/lib.rs`

Reply:
@drknzz #919 merged already, feel free to change stuff in the files in mentioned